README.md

```diff
@@ -1,43 +1,43 @@
-## CloudFoundry PHP Example Application: Drupal 
+## cloud.gov example application: Drupal 7
 
-This is an example application which can be run on CloudFoundry using the [PHP Build Pack].
+This is an example application that can be run on [cloud.gov](https://www.cloud.gov) using the [PHP Buildpack](http://docs.cloudfoundry.org/buildpacks/php/index.html).
 
-This is an out-of-the-box implementation of Drupal.  It's an example of how common PHP applications can easily be run on CloudFoundry.
+This is an out-of-the-box implementation of Drupal 7.  It's an example of how common PHP applications can easily be run on cloud.gov.
 
 ### Usage
 
 1. Clone the app (i.e. this repo).
 
-  ```bash
-  git clone https://github.com/cloudfoundry-samples/cf-ex-drupal.git cf-ex-drupal
-  cd cf-ex-drupal
-  ```
+    ```bash
+    git clone https://github.com/18F/cf-ex-drupal.git cf-ex-drupal
+    cd cf-ex-drupal
+    ```
```

Comment:
we need to add an instruction to change the hostname of the app in the manifest.yml. upon pushing at this point, i received the error:

```
Creating app cf-ex-drupal in org sandbox-gsa / space jamesc.scott as jamesc.scott@gsa.gov...
OK

Using route cf-ex-drupal.app.cloud.gov
Binding cf-ex-drupal.app.cloud.gov to cf-ex-drupal...
FAILED
The route cf-ex-drupal.app.cloud.gov is already in use.
TIP: Change the hostname with -n HOSTNAME or use --random-route to generate a new route and then push again.
```